src/Metadata/MetadataFactory.php

```diff
@@ -16,36 +16,36 @@
 class MetadataFactory
 {
     /**
-     * @var string[]
+     * @var array<string, int|string>
      *
-     * @phpstan-var class-string[]
+     * @phpstan-var array<class-string, int|string>
      */
     private $auditedEntities = [];
 
     /**
-     * @phpstan-param array<class-string, mixed> $auditedEntities
+     * @phpstan-param class-string[] $auditedEntities
      */
     public function __construct(array $auditedEntities)
     {
-        $this->auditedEntities = array_flip(array_filter($auditedEntities, static function ($record): bool {
-            return \is_string($record) || \is_int($record);
-        }));
+        $this->auditedEntities = array_flip($auditedEntities);
```

Comment:
Apparently this was done in https://github.com/sonata-project/EntityAuditBundle/issues/143 because of SonataDoctrineORMAdminBundle, but [looking at the tests there](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1652), looks like there are no `null` values. We could still add an `array_filter` call just in case, but if there are `null` values, we should fix it in the doctrine orm admin bundle.